sdk/src/feature_set.rs

```diff
@@ -500,8 +500,8 @@ pub mod return_none_for_zero_lamport_accounts {
     solana_sdk::declare_id!("7K5HFrS1WAq6ND7RQbShXZXbtAookyTfaDQPTJNuZpze");
 }
 
-pub mod on_load_preserve_rent_epoch_for_rent_exempt_accounts {
```

Comment:
can we retain the feature for the sake of listing in CLI?

Reply:
done